include/toplevel.mk

```diff
@@ -213,6 +213,7 @@ check: .config FORCE
 	@+$(NO_TRACE_MAKE) -r -s $@ QUIET= V=s
 
 val.%: FORCE
+var.%: FORCE
```

Comment:
Why not merge these as noted before? Doesn't this break the `val.%` target?

Reply:
yeah you're right, I don't know why I thought I had done something like that before...